doc/WebRTC.xml

```diff
@@ -672,6 +694,38 @@ Server -> Device: { "method": "trickle", "params": {"session": "s1", "candidate"
       <para>For two-way audio it's recommended to use echo cancellation to avoid feedback
         loops.</para>
     </section>
+    <section xml:id="section_data_channels">
+      <title>Data channels</title>
+        <section xml:id="section_data_channels_enable">
+          <title>Enabling data channels</title>
+            <para>If the device supports WebRTC data channels, it shall signal this in the SDP offer 
+              of the invite method. If the client supports data channels and intends to use them during 
+              the WebRTC session, it shall signal this in the SDP answer.</para>
+            <para>ONVIF defines data channel subprotocol names starting with the “vnd.onvif” prefix. 
+              Usage of this prefix is restricted to ONVIF-defined subprotocols only.</para>
+            <para>Before creating and using any data channel subprotocol, the device and client need 
+              to exchange a list of allowed data channel subprotocols in the invite method. Only 
+              subprotocols provided by both the device and client during the invite method shall be 
+              allowed to be enabled during the WebRTC session.</para>
+            <para>At any time after obtaining a WebRTC connection, a client may start using a data 
+              channel subprotocol by creating a new data channel with the specified subprotocol 
+              identifier. When a new data channel event is triggered on a device, the device should 
+              enable the relevant data channel subprotocol.

```

Comment:
If a subprotocol is attempted to be used by a client, but is not allowed (as per the initial handshakes), what should the device do? And how to verify in a test tool that the refusal works as expected? If the data channel remains open, but no data is sent by the device, should the verification step involve some kind of timeout limit before the test passes? It would be cleaner and more direct if the device instead actively closes the data channel? Or simply closes the whole peer-to-peer session?

Reply:
I think closing the data channel explicitly by the device makes most sense, as the client will also get the feedback. I will add such requirement if that's OK for everyone.

Reply:
fine with me. And again it's no big deal, just thought it could simplify verification